zerver/models.py

```diff
@@ -1684,7 +1684,7 @@ class UserProfile(AbstractBaseUser, PermissionsMixin, UserBaseSettings):
     # provide a unique key for a user for the mobile push
     # notifications bouncer that will not have collisions after doing
     # a data export and then import.
-    uuid: Optional[UUID] = models.UUIDField(null=True, default=uuid4, unique=True)
+    uuid: UUID = models.UUIDField(default=uuid4, unique=True)
```

Comment:
Might be worth a note in the commit message that this is a separate commit to ensure that the backfill in the previous commit runs, so there are no nulls by the time we make this non-optional.

Reply:
Yeah, agreed, that'd be very helpful.

Reply:
True - added a note on that